specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json

```diff
@@ -1472,6 +1472,75 @@
         "entryFilePath"
       ]
     },
+    "ExecuteSSISPackageActivity": {
+      "description": "Execute SSIS package activity.",
+      "x-ms-discriminator-value": "ExecuteSSISPackage",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ExecutionActivity"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "x-ms-client-flatten": true,
+          "description": "Execute SSIS package activity properties.",
+          "$ref": "#/definitions/ExecuteSSISPackageActivityTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "ExecuteSSISPackageActivityTypeProperties": {
+      "description": "Execute SSIS package activity properties.",
+      "properties": {
+        "packageLocation": {
+          "description": "SSIS package location.",
+          "$ref": "#/definitions/SSISPackageLocation"
+        },
+        "use32bitRuntime": {

```

Comment:
I'm surprised to see that the service takes this as a 'Yes' or 'No' string, and not a bool. Any particular reason that isn't the approach?

Reply:
@ravbhatnagar Please share more comments on this. I remembered you suggested this practice in one of my CR. Thanks.

Reply:
bools are discouraged. however a string which takes yes or no as input is not any better. The idea is to make the property values more descriptive and allow for expansion of allowed values in future if needed.

Reply:
Alright, thanks for the context here everyone. @zhangyd2015, I think there are two options on this property moving forward then:

1. (Preferred) Update this property to be called something like `runtime` and have the enum be `x64`, `x86`, or whatever else comes along for available runtime.
1. Move this back to being a `bool`. (Though, it sounds like this is discouraged.)